cli/internal/cmd/root.go

```diff
@@ -25,10 +25,21 @@ var (
 
 func recovery() {
 	if r := recover(); r != nil {
-		fmt.Println(emoji.Sprintf("\n:skull:Oops! Unexpected error occurred. Please contact tech support.\n"))
+		// Always log to file
 		log.Error("panic detected", "error", r)
 		log.Error(string(debug.Stack()))
 
+		// If debug mode is enabled, show detailed panic info in console and exit
+		if viper.GetBool("debug") {
```

Comment:
nit: this hasn't been followed consistently in all places where we access config parameters, but we want to move away from accessing them directly from viper, and use the config struct instead. Ideally we should do:
```
config.GetConfig().Debug
```

Maybe this is a good opportunity to fix it in other places as well where we use viper outside the config package